CHANGELOG.md

```diff
@@ -2,9 +2,17 @@
 
 All notable changes to this project will be documented in this file.
 
-## [v2.4.0](https://github.com/voxpupuli/metadata-json-lint/tree/v2.4.0) (2020-06-12)
+## [v2.5.0](https://github.com/voxpupuli/metadata-json-lint/tree/v2.5.0) (2020-11-17)
 
-[Full Changelog](https://github.com/voxpupuli/metadata-json-lint/compare/2.3.0...v2.4.0)
+[Full Changelog](https://github.com/voxpupuli/metadata-json-lint/compare/2.4.0...v2.5.0)
+
+**Merged pull requests:**
+
+- Require Ruby 2.1 and drop post\_install\_message [\#112](https://github.com/voxpupuli/metadata-json-lint/pull/112) ([ekohl](https://github.com/ekohl))
```

Comment:
Is this a minor release? Not sure how much pins this module and how painful a major release is.

Reply:
we could also do a major release, I just wasn't 100% sure about it.

Reply:
I'm leaning to a major release since it's dropping an older Ruby version.

Reply:
updated it